CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 6.2.2
+
+- fixed wrong property in `fail2ban_jail` and `fail2ban_filter` resources
+- added documentation for above changes
+
 ## Unreleased
```

Comment:
This doesn't look right. The top one should be `## Unreleased`.

Reply:
> This doesn't look right. The top one should be `## Unreleased`.

Yeah fwiw - wasn't sure the order. Let me change.

Reply:
pushed vvvv